jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java

```diff
@@ -58,22 +58,40 @@ public class Dispatcher {
 
     /**
      * Handle an HTTP request if it is sent to a registered dataset.
-     *
+     * <p>
      * Fuseki uses dynamic dispatch, the set of registered datasets can change while
      * the server is running, so dispatch is driven off Fuseki system registries.
-     *
+     * <p>
      * If the request URL matches a registered dataset, process the request, and send
      * the response.
-     *
+     * <p>
      * This function is called by {@link FusekiFilter#doFilter}.
-     *
+     * <p>
      * Returns {@code true} if the request has been handled, including an error response sent,
      * and returns false (no error or response sent) if the request has not been handled.
-     *
+     * <p>
      * This function does not throw exceptions.
+     * <p>
+     * The dispatch process is:
+     * <ul>
+     * <li> Decide the data, based on the request URI ({@link #locateDataAccessPoint(HttpServletRequest, DataAccessPointRegistry)}
+     * <li> Allocate HttpAction ({@link #process}, {@link #dispatchAction}).
+     * <li> Decide service endpoint name ({@link #chooseProcessor})
+     * <li> Decvice operation {@link #chooseEndpoint}
```

Comment:
s/Decvice/Decide (also not sure if the space between `<li>` and text matter, probably it doesn't)